akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/HttpChallenge.scala

```diff
@@ -24,6 +25,8 @@ final case class HttpChallenge(scheme: String, realm: String,
 
 object HttpChallenges {
 
+  def negotiate(): HttpChallenge = HttpChallenge("Negotiate", "")
+
   def basic(realm: String): HttpChallenge = HttpChallenge("Basic", realm)
 
   def oAuth2(realm: String): HttpChallenge = HttpChallenge("Bearer", realm)
```

Comment:
I've previously voted to get rid of these and will do so again here if there's not yet a release containing these constructors and nobody disagrees. :)